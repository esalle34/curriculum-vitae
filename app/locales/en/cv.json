{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Profile",
  "description": "<0>As frontend software engineer since 2017</0>, i bring new nice user interface features and do the maintenance of your websites and apps.",
  "qualities": "<0>Curious and always seeking new opportunities to learn</0>, i like to communicate and bring new skills to my team members.",
  "remote": "Furthermore, having experience as remote software engineer, i am able to work with appreciable <0>autonomy</0> as a remote team member.",
  "altaprofitsTitle": "Altaprofits <0>March 2017- March 2020</0>",
  "altaprofitsIntroduction": "Within a team of 3 people, maintenance and development of front-end interfaces.",
  "altaprofitsWork": [
    "Setting up the React library",
    "New frontend features",
    "New back-office features",
    "Back office management",
    "Site maintenance"
  ],
  "wefightTitle": "Wefight <0>April 2021 - October 2022</0>",
  "wefightIntroduction": "With the front-end chapter made up of 5 people, design of interfaces for the back office as well as a patient monitoring platform.",
  "wefightWork": [
    "Design and integration of frontend interfaces in React - Redux",
    "Development on a Backend for frontend in Express",
    "Management of auth0 rights",
    "Implementation from a mongoDB database",
    "Typescript",
    "Unit tests, integration tests"
  ],
  "24sTitle": "24s (LVMH group) <0>October 2022 - ...</0>",
  "24sIntroduction": "Within the web chapter made up of 4 people, maintenance and development of new features for the 24s e-commerce platform.",
  "24sWork": [
    "Frontend interface design in React - Redux",
    "Development on a Next base",
    "Intervention within several feature teams",
    "Integration of design from figma models",
    "Typescript",
    "Unit tests, integration tests"
  ],
  "hardSkills": "Skills",
  "languages": "Languages",
  "french": "French",
  "english": "English",
  "spanish": "Spanish",
  "contact": "Contact me"
}
