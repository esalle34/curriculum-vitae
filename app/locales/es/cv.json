{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Perfil",
  "description": "<0>Desarrollador front-end desde 2017</0>, creo funcionalidades y mantengo tus sitios y aplicaciones.",
  "qualities": "<0>Curioso y siempre buscando novedades</0>, disfruto comunicándome e introduciendo a mis colegas nuevos métodos y nuevas herramientas.",
  "remote": "Por fin, gracias a mi experiencia remota, puedo responder a sus solicitudes a distancia <0>con total autonomía</0>.",
  "altaprofitsTitle": "Altaprofits <0>Marzo 2017- Marzo 2020</0>",
  "altaprofitsIntroduction": "Dentro de un equipo de 3 personas, mantenimiento y desarrollo de interfaces front-end.",
  "altaprofitsWork": [
    "Configurando la biblioteca React",
    "Nuevas funcionalidades de la interfaz",
    "Nuevas funcionalidades del back-office",
    "Gestión de back office",
    "Mantenimiento del sitio"
  ],
  "wefightTitle": "Wefight <0>Abril 2021 - Octubre 2022</0>",
  "wefightIntroduction": "Con el capítulo front-end conformado por 5 personas, diseño de interfaces para el back office así como una plataforma de seguimiento de pacientes.",
  "wefightWork": [
    "Diseño e integración de interfaces frontend en React - Redux",
    "Desarrollo en un Backend para frontend en Express",
    "Gestión de derechos auth0",
    "Implementación desde una base de datos mongoDB",
    "Typescript",
    "Pruebas de integración, pruebas de unidad"
  ],
  "24sTitle": "24s (grupo LVMH) <0>Octubre 2022 - ...</0>",
  "24sIntroduction": "Dentro del capítulo web conformado por 4 personas, mantenimiento y desarrollo de nuevas funcionalidades para la plataforma de comercio electrónico 24s.",
  "24sWork": [
    "Diseño de interfaz frontend en React - Redux",
    "Desarrollo sobre una base Next",
    "Intervención en varios equipos",
    "Integración del diseño de los modelos figma",
    "Typescript",
    "Pruebas de integración, pruebas de unidad"
  ],
  "hardSkills": "Habilidades",
  "languages": "Idiomas",
  "french": "Francés",
  "english": "Inglés",
  "spanish": "Español",
  "contact": "Contactar"
}
