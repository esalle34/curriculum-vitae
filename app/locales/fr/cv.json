{
  "title": "Curriculum Vitae - Eric Salle",
  "profil": "Profil",
  "description": "<0>Développeur front-end depuis 2017</0>, je crée des fonctionnalités et réalise la maintenance de vos sites et applications.",
  "qualities": "<0>Curieux et toujours à la recherche de la nouveauté</0>, j'apprécie communiquer et faire découvrir à mes collègues de nouvelles méthodes et de nouveaux outils.",
  "remote": "Enfin, fort d'une expérience en remote je suis capable de pourvoir à vos demandes à distance <0>en toute autonomie</0>.",
  "altaprofitsTitle": "Altaprofits <0>Mars 2017- Mars 2020</0>",
  "altaprofitsIntroduction": "Au sein d'une équipe de 3 personnes, maintenance et développement des interfaces front-end.",
  "altaprofitsWork": [
    "Mise en place de la librairie React",
    "Nouvelles fonctionnalités frontend",
    "Nouvelles fonctionnalités back-office",
    "Gestion du back-office",
    "Maintenance du site"
  ],
  "wefightTitle": "Wefight <0>Avril 2021 - Octobre 2022</0>",
  "wefightIntroduction": "Avec le chapitre front-end constitué de 5 personnes, conception d'interfaces à destination du back-office ainsi que d'une plateforme de suivi patient.",
  "wefightWork": [
    "Conception et intégration d'interfaces frontend en React - Redux",
    "Développement sur un Backend for frontend en Express",
    "Gestion des droits auth0",
    "Mise en place à partir d'une base mongoDB",
    "Typescript",
    "Tests unitaire, tests d'intégration"
  ],
  "24sTitle": "24s (LVMH group) <0>Octobre 2022 - ...</0>",
  "24sIntroduction": "Au sein du chapitre web constitué de 4 personnes, maintenance et développement de nouvelles fonctionnalités pour la plateforme e-commerce de 24s.",
  "24sWork": [
    "Conception d'interfaces frontend en React - Redux",
    "Développement sur un socle en Next",
    "Intervention au sein de plusieurs feature team",
    "Integration du design à partir de maquettes figma",
    "Typescript",
    "Tests unitaire, tests d'intégration"
  ],
  "hardSkills": "Compétences",
  "languages": "Langues",
  "french": "Français",
  "english": "Anglais",
  "spanish": "Espagnol",
  "contact": "Contact"
}
