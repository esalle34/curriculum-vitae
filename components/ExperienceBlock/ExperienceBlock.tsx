import { decodeTranslation } from "@/app/locales/helpers";
import parse from "html-react-parser";

interface Props {
  title: string;
  introduction: string;
  work: ReadonlyArray<string>;
}

const ExperienceBlock = ({ title, introduction, work }: Props) => {
  return (
    <section className="flex flex-col border-2 rounded-xl shadow-md">
      <h3 className="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center">
        {parse(
          decodeTranslation({
            translation: title,
            options: {
              htmlTag: "span",
              className: "pl-4 text-base text-blue-500",
            },
          })
        )}
      </h3>
      <p className="text-blue-500 px-4 leading-8">{introduction}</p>
      <ul className="p-4 md:pl-8">
        {work.map((line, index) => (
          <li className="text-gray-600 leading-8 text-sm" key={`work-${index}`}>
            {line}
          </li>
        ))}
      </ul>
    </section>
  );
};

export default ExperienceBlock;
