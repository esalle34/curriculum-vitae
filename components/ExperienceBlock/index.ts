import ExperienceBlock from "./ExperienceBlock";

export default ExperienceBlock;
