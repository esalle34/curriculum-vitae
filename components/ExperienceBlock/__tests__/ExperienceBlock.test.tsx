import { render } from "@testing-library/react";
import ExperienceBlock from "../ExperienceBlock";
import * as dict from "@/app/locales/fr/cv.json";
import * as dictEN from "@/app/locales/en/cv.json";
import * as dictES from "@/app/locales/es/cv.json";

describe("ExperienceBlock", () => {
  it("should match Altaprofits translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dict.altaprofitsTitle}
        introduction={dict.altaprofitsIntroduction}
        work={dict.altaprofitsWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match Wefight translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dict.wefightTitle}
        introduction={dict.wefightIntroduction}
        work={dict.wefightWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match 24s translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dict["24sTitle"]}
        introduction={dict["24sIntroduction"]}
        work={dict["24sWork"]}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match Altaprofits EN translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictEN.altaprofitsTitle}
        introduction={dictEN.altaprofitsIntroduction}
        work={dictEN.altaprofitsWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match Wefight EN translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictEN.wefightTitle}
        introduction={dictEN.wefightIntroduction}
        work={dictEN.wefightWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match 24s EN translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictEN["24sTitle"]}
        introduction={dictEN["24sIntroduction"]}
        work={dictEN["24sWork"]}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match Altaprofits ES translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictES.altaprofitsTitle}
        introduction={dictES.altaprofitsIntroduction}
        work={dictES.altaprofitsWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match Wefight ES translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictES.wefightTitle}
        introduction={dictES.wefightIntroduction}
        work={dictES.wefightWork}
      />
    );
    expect(container).toMatchSnapshot();
  });
  it("should match 24s ES translations snapshot", () => {
    const { container } = render(
      <ExperienceBlock
        title={dictES["24sTitle"]}
        introduction={dictES["24sIntroduction"]}
        work={dictES["24sWork"]}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
