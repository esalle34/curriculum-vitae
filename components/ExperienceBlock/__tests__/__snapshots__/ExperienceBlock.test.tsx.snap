// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExperienceBlock should match 24s EN translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      24s (LVMH group) 
      <span
        class="pl-4 text-base text-blue-500"
      >
        October 2022 - ...
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Within the web chapter made up of 4 people, maintenance and development of new features for the 24s e-commerce platform.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Frontend interface design in React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Development on a Next base
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Intervention within several feature teams
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Integration of design from figma models
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Unit tests, integration tests
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match 24s ES translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      24s (grupo LVMH) 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Octubre 2022 - ...
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Dentro del capítulo web conformado por 4 personas, mantenimiento y desarrollo de nuevas funcionalidades para la plataforma de comercio electrónico 24s.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Diseño de interfaz frontend en React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Desarrollo sobre una base Next
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Intervención en varios equipos
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Integración del diseño de los modelos figma
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Pruebas de integración, pruebas de unidad
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match 24s translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      24s (LVMH group) 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Octobre 2022 - ...
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Au sein du chapitre web constitué de 4 personnes, maintenance et développement de nouvelles fonctionnalités pour la plateforme e-commerce de 24s.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Conception d'interfaces frontend en React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Développement sur un socle en Next
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Intervention au sein de plusieurs feature team
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Integration du design à partir de maquettes figma
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Tests unitaire, tests d'intégration
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Altaprofits EN translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Altaprofits 
      <span
        class="pl-4 text-base text-blue-500"
      >
        March 2017- March 2020
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Within a team of 3 people, maintenance and development of front-end interfaces.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Setting up the React library
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        New frontend features
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        New back-office features
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Back office management
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Site maintenance
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Altaprofits ES translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Altaprofits 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Marzo 2017- Marzo 2020
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Dentro de un equipo de 3 personas, mantenimiento y desarrollo de interfaces front-end.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Configurando la biblioteca React
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Nuevas funcionalidades de la interfaz
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Nuevas funcionalidades del back-office
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Gestión de back office
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Mantenimiento del sitio
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Altaprofits translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Altaprofits 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Mars 2017- Mars 2020
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Au sein d'une équipe de 3 personnes, maintenance et développement des interfaces front-end.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Mise en place de la librairie React
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Nouvelles fonctionnalités frontend
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Nouvelles fonctionnalités back-office
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Gestion du back-office
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Maintenance du site
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Wefight EN translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Wefight 
      <span
        class="pl-4 text-base text-blue-500"
      >
        April 2021 - October 2022
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      With the front-end chapter made up of 5 people, design of interfaces for the back office as well as a patient monitoring platform.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Design and integration of frontend interfaces in React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Development on a Backend for frontend in Express
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Management of auth0 rights
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Implementation from a mongoDB database
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Unit tests, integration tests
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Wefight ES translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Wefight 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Abril 2021 - Octubre 2022
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Con el capítulo front-end conformado por 5 personas, diseño de interfaces para el back office así como una plataforma de seguimiento de pacientes.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Diseño e integración de interfaces frontend en React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Desarrollo en un Backend para frontend en Express
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Gestión de derechos auth0
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Implementación desde una base de datos mongoDB
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Pruebas de integración, pruebas de unidad
      </li>
    </ul>
  </section>
</div>
`;

exports[`ExperienceBlock should match Wefight translations snapshot 1`] = `
<div>
  <section
    class="flex flex-col border-2 rounded-xl shadow-md"
  >
    <h3
      class="flex text-blue-800 font-bold italic p-4 text-2xl mb-4 border-b-2 justify-between items-center"
    >
      Wefight 
      <span
        class="pl-4 text-base text-blue-500"
      >
        Avril 2021 - Octobre 2022
      </span>
    </h3>
    <p
      class="text-blue-500 px-4 leading-8"
    >
      Avec le chapitre front-end constitué de 5 personnes, conception d'interfaces à destination du back-office ainsi que d'une plateforme de suivi patient.
    </p>
    <ul
      class="p-4 md:pl-8"
    >
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Conception et intégration d'interfaces frontend en React - Redux
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Développement sur un Backend for frontend en Express
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Gestion des droits auth0
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Mise en place à partir d'une base mongoDB
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Typescript
      </li>
      <li
        class="text-gray-600 leading-8 text-sm"
      >
        Tests unitaire, tests d'intégration
      </li>
    </ul>
  </section>
</div>
`;
